src/test/java/uk/gov/hmcts/reform/sscs/controller/CcdMideventCallbackControllerTest.java

```diff
@@ -358,5 +367,191 @@ public void handleCcdMidEventAdminRestoreCasesWhenPathIsExtractedAndFailuresRetu
             .andExpect(content().json(expectedJsonErrorsAndWarningsString));
     }
 
+    @Test
+    public void handleCcdMidEventAdjournCaseDirectionDueDate_EmptyDueDateReturnsError() throws Exception {
+
+        String path = getClass().getClassLoader().getResource("sya/allDetailsForGeneratePdf.json").getFile();
+        String content = FileUtils.readFileToString(new File(path), StandardCharsets.UTF_8.name());
+
+        SscsCaseData sscsCaseData = SscsCaseData.builder()
+                .adjournment(Adjournment.builder().build())
+                .build();
+
+        when(deserializer.deserialize(content)).thenReturn(new Callback<>(
+                new CaseDetails<>(ID, JURISDICTION, State.HEARING, sscsCaseData, LocalDateTime.now(), "Benefit"),
+                Optional.empty(), ADJOURN_CASE, false));
```

Comment:
since the steps above are used by most tests, it might be worth moving them to a method